homeassistant/bootstrap.py

```diff
@@ -606,6 +606,7 @@ async def async_enable_logging(
     logging.getLogger("urllib3").setLevel(logging.WARNING)
     logging.getLogger("aiohttp.access").setLevel(logging.WARNING)
     logging.getLogger("httpx").setLevel(logging.WARNING)
+    logging.getLogger("async_upnp_client.profiles.profile").setLevel(logging.WARNING)
```

Comment:
Did you discuss that with @StevenLooman ? It might be better to adjust the library directly if it is too verbose.